src/proto/crypt.pro

```diff
@@ -30,5 +30,9 @@ void crypt_sodium_lock_key(char_u *key);
 int crypt_sodium_munlock(void *const addr, const size_t len);
 void crypt_sodium_randombytes_buf(void *const buf, const size_t size);
 int crypt_sodium_init(void);
+#ifdef VMS
+UINT32_T crypt_sodium_randombytes_random(void);
+#else
 uint32_t crypt_sodium_randombytes_random(void);
+#endif
```

Comment:
`crypt.pro` is automatically generated by using cproto. This change will be discarded next time.

Reply:
Thank you for the information.
What would be the correct way to implement this change?

Reply:
I think we can simply use UINT32_T here (and in crypt.c).

Reply:
Thank you. 
I have just committed the change in this pull request.